Adding new WW2 units: 101st Airborine and Generic HEER Units

Dennisz125 opened the pull request:
I followed the instructions as best as I could. I hope the inheritance is solid and there's nothing wrong with it.

Dennisz125 committed:
Add a new WW2 units: 101st Airborine and Generic HEER Units

Dennisz125 committed:
Update Files and Remove unnecessary files

Til-Weimann reviewed (changes requested):
This PR modifies the GC_NVA base group instead of creating a new one
Faction inheritance should be done from scratch, also you should not call it Heer but Wehrmacht (GC_WM), you are also missing a faction config
US 1942 characters should also inherit from Character_US_Base and not BaseLoadout, they should not be based on the Cold War US loadout after all
I also saw a reference to what seems like a custom MG42 which is not included in the PR
There may be more issues, I only had a glance

Dennisz125 committed:
Merge branch 'master' into WW2-Units

Dennisz125 committed:
Fixes based on Til's feedback

- Moved the custom backpack and weapons from my Prefab folder to GC Units' Prefab folder
- Update 1945 US characters' inherit from Character_US_BaseLoutout to Character_US_Base
- Rename the HEER files to WM
- Double check that any changes to any other factions shouldn't effect the CG_WM outside of the intended future updates.
- Made and add a new Faction config file to the FactionManager

Dennisz125 committed:
Rename the Units, moved prefabs, and add a flag for the Faction

Dennisz125 committed:
Update the GC_WM with a new vest

TheLocalPub commented:
Might as well just close this....

Dennisz125 commented:
> Might as well just close this....

Why are we closing if the branch is ready to be merged?

Dennisz125 commented:
Hold that thought, after updating the units. Some things are a bit out of sync. I'll get them fixed and force @Til-Weimann to review them again for the completed WW2 missions.
